Library/test-helpers/lib.sh

```diff
@@ -2161,6 +2161,58 @@ limeconRunRegistrar() {
 true <<'=cut'
 =pod
 
+=head2 limeconRunSystemd
+
+Container run via podman with specified parameters.
+
+    limeconRunSystemd NAME TAG IP NETWORK COMMAND EXTRA_PODMAN_ARGS
+
+=item NAME
+
+Set name of container.
+
+=item TAG
+
+Name of image tag.
+
+=item IP
+
+IP address of container.
+
+=item NETWORK
+
+Name of used podman network.
+
+=item COMMAND
+
+Specify running command at start of container.
+
+=item EXTRA_PODMAN_ARGS
+
+Specify setup of starting container.
+
+=back
+
+Returns 0.
+
+=cut
+
+limeconRunSystemd() {
+
+    local NAME=$1
+    local TAG=$2
+    local IP=$3
+    local NETWORK=$4
+    local COMMAND=$5
+    local EXTRA_PODMAN_ARGS=$6
+
+    limeconRun $NAME $TAG $IP $NETWORK $COMMAND $EXTRA_PODMAN_ARGS
```

Comment:
I would expect you would be running `/sbin/init` here, instead of passing the command. And that there won't be any `COMMAND` in arguments.